### PR TITLE
merging project top and project subtomograms into project subtomograms

### PR DESCRIPTION
**Pull request by Vilax**

### CODE REVIEW

**Comment by albertmena** [comment]
Fix the test please

File: xmipptomo/protocols/protocol_project_top.py [hunk 1]
```diff
@@ -154,7 +154,7 @@ def createOutputStep(self):
         # Input could be SetOfVolumes or SetOfSubtomograms
         for item in input.iterItems():
             idx = item.getObjId()
-            p = Particle()
+            p = Particle(objId=item.getObjId())
```
Comment (MartinSalinas98) [inline xmipptomo/protocols/protocol_project_top.py:157]: If project top is getting absorbed by project subtomograms, shouldn't this protocol get deleted?

**Comment by MartinSalinas98** [comment]
I need to properly understand the new features included in the protocol before i can really check this code.

**Comment by MartinSalinas98** [comment]
Not needed any more, can be replaced by #196.

In such Pull Request, you can project in whichever axis you want just setting your `NSamples` to `1` and adding the same value to tilt range start and end.

For example, to proyect on to Z axis:
- NSamples: 1
- Tilit range start: 0
- Tilt range end: 0

**Pull request closed without merging**